Assets/HoloToolkit-Examples/SpatialMapping/Scripts/PlaneFindingTest.cs

```diff
@@ -111,7 +111,7 @@ private void OnDrawGizmosSelected()
                     UnityEditor.Handles.DrawLine(corners[1], corners[2]);
                     UnityEditor.Handles.DrawLine(corners[1], corners[3]);
                     UnityEditor.Handles.DrawLine(corners[2], corners[3]);
-                    UnityEditor.Handles.ArrowHandleCap(0, center, Quaternion.FromToRotation(Vector3.forward, normal), 0.4f, EventType.ignore);
+                    UnityEditor.Handles.ArrowHandleCap(0, center, Quaternion.FromToRotation(Vector3.forward, normal), 0.4f, EventType.Ignore);
```

Comment:
Should probably revert this change. We're not using the 2017.3 Editor for the dev branch.